client/navbar.jsx

```diff
@@ -0,0 +1,106 @@
+import axios from 'axios';
+import React from 'react';
+
+export default class Navbar extends React.Component {
+  constructor(){
+    super();
+    this.state = {
+      displaySort: false,
+      displayGenre: false,
+      displayHamburger: false,
+      genres: []
+    }
+  }
+
+  handleClick = () => {
+    this.setState({ displayHamburger: !this.state.displayHamburger });
+  }
+
+  handleDropDown = (e) => {
+    this.setState({ displaySort: !this.state.displaySort })
+  }
+
+  handleGenre = () => {
+    this.setState({ displayGenre: !this.state.displayGenre })
+  }
+
+  handleGenreSelection = (id) => {
+    this.setState({displayHamburger: false});
+    this.props.handleGenreSelection(id);
+  }
+
+  handleSortSelection = (sortby) => {
+    this.setState({ displayHamburger: false });
+    this.props.handleSortSelection(sortby)
+  }
+
+  componentDidMount() {
+    axios.get('http://localhost:3001/getGenres')
```

Comment:
You should be able to just say `/getGenres`. Then you won't need that `cors` middleware anymore